<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadingPeriodWidget</class>
 <widget class="QWidget" name="ReadingPeriodWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="p_checkIsCyclick">
     <property name="text">
      <string>Cyclic</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="p_spinBoxPeriod">
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="p_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
